production.ini

```diff
@@ -204,9 +204,7 @@ stacks_enabled = False
 # updates.  Users have free-reign to override them for each kind of entity.  At
 # the end of the day, we only consider the requirements defined by single
 # updates themselves when gating in the backend masher process.
-site_requirements = depcheck upgradepath
-## Some day we'll have rpmgrill, and that will be cool.  Ask tflink.
-#site_requirements = depcheck upgradepath rpmgrill
+site_requirements = dist.rpmdeplint dist.upgradepath
```

Comment:
I've got a PR open to rework basically everything about how settings in Bodhi work: #1544. If that gets merged first, we should make sure we make these changes in the new ```bodhi.server.config.BodhiConfig._defaults```. If not, I'll have to be sure to add this there.